src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -60,7 +60,7 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
to maintain consistency, check how email and phone values are obtained, do the same with address. 

PS: better way to ask the question;
- i tried X, Y, Z which one is better? 
rather than 
- are there other ways about this?  ;-)